packages/edit-post/src/components/layout/index.js

```diff
@@ -310,7 +310,8 @@ function Layout( { initialPost } ) {
 				editorNotices={ <EditorNotices /> }
 				secondarySidebar={ secondarySidebar() }
 				sidebar={
-					( ! isMobileViewport || sidebarIsOpened ) && (
+					( ( isMobileViewport && sidebarIsOpened ) ||
+						( ! isMobileViewport && ! isDistractionFree ) ) && (
```

Comment:
This is here because the open sidebar action no longer toggles DFM off.

Reply:
yes